src/scalarstats.jl

```diff
@@ -197,75 +194,117 @@ nquantile(v::AbstractArray{<:Real}, n::Integer) = quantile(v, (0:n)/n)
 """
     span(x)
 
-Return the span of an integer array, i.e. the range `minimum(x):maximum(x)`.
-The minimum and maximum of `x` are computed in one-pass using `extrema`.
+Return the span of a collection, i.e. the range `minimum(x):maximum(x)`.
+The minimum and maximum of `x` are computed in one pass using `extrema`.
 """
-span(x::AbstractArray{<:Integer}) = ((a, b) = extrema(x); a:b)
+span(x) = ((a, b) = extrema(x); a:b)
 
 # Variation coefficient: std / mean
 """
     variation(x, m=mean(x))
 
-Return the coefficient of variation of an array `x`, optionally specifying
+Return the coefficient of variation of collection `x`, optionally specifying
 a precomputed mean `m`. The coefficient of variation is the ratio of the
 standard deviation to the mean.
 """
-variation(x::AbstractArray{<:Real}, m::Real) = stdm(x, m) / m
-variation(x::AbstractArray{<:Real}) = variation(x, mean(x))
+variation(x, m) = stdm(x, m) / m
+variation(x) = ((m, s) = mean_and_std(x); s/m)
```

Comment:
```suggestion
variation(x) = \(mean_and_std(x)...)
```

This is is a bit gross, but I still think less gross than multistatement single-liners

Reply:
Why? :-/

Reply:
too many equals  signs.
Though feel free to disagree

Reply:
FWIW I prefer what's written currently.